tests/loki/iofq_test.py

```diff
@@ -98,10 +121,11 @@ def test_pipeline_can_compute_IofQ(uncertainties, qxy: bool):
         BackgroundSubtractedIofQxy,
     ],
 )
-def test_pipeline_can_compute_IofQ_in_event_mode(uncertainties, target):
-    pipeline = make_workflow()
+def test_pipeline_can_compute_IofQ_in_event_mode(
```

Comment:
I think the approach is fine. The test looks almost identical to before, it just has a fixture which has some pre-computed stuff in it. It's not making some tests depend on others or anything like that.

How does it change the overall time it takes to run all tests?

Reply:
> How does it change the overall time it takes to run all tests?

You would have to do something similar for quite a few more tests. But the worse offender is currently the beam-center finder, but I don't feel it is worth playing with that.